### PR TITLE
Make the badge tool logos clickable like titles

### LINKED ISSUE #277: Test detail score badge logos should link in page like the tool titles do
Screen Shot 2020-05-18 at 7 56 23 AM

### PR DESCRIPTION
**Pull request by pajaydev**
Fixes https://github.com/GoogleChromeLabs/tooling.report/issues/277

### CODE REVIEW

**Comment by pajaydev** [comment]
@googlebot I signed it!

**Review by argyleink — CHANGES_REQUESTED** [review]
Hi @pajaydev, thanks for taking this on! 

This does cause a regression in shape, the images are no longer circles! 🙂 

<details>

<summary>screenshot</summary>

Screen Shot 2020-07-02 at 10 22 53 AM

</details>

Looks like making the image a link has added padding and changed the box shape. I added `padding: 0;` to the `.toolIcon` class in this file `static-build/components/TestResultSnippet/styles.css` and it resolved the issue. Can you update this PR to remove the padding from toolIcons please 👍 Then I think this is g2g.

**Comment by pajaydev** [comment]
@argyleink  Thanks 👍  I have reset the padding. Kindly review and check the below screenshot


Screen Shot 2020-07-02 at 1 33 24 PM

**Comment by surma** [comment]
(@developit unrelated, any idea why the preview script isn’t running on this one?)

**Comment by developit** [comment]
@surma I think it only runs on non-fork PRs, because it's not an official GH app.

File: static-build/components/TestResultSnippet/index.tsx [hunk 1]
```diff
@@ -28,8 +28,13 @@ const TestResultSnippet: FunctionalComponent<Props> = ({
       <a href={`#${name}`} title={`Explain the ${result}`} class={$toolName}>
         {name}
       </a>
-      <figure class={$toolIcon}>
-        <img src={toolImages[name as BuildTool]} alt="" />
+      <figure>
+        <a href={`#${name}`} class={$toolIcon}>
+          <img
+            src={toolImages[name as BuildTool]}
+            alt={`${name} bundler logo`}
```
Comment (jakearchibald) [inline static-build/components/TestResultSnippet/index.tsx]:
```suggestion
            alt={`${name}`}
```

Comment (jakearchibald) [reply static-build/components/TestResultSnippet/index.tsx]: Minor tweak. In terms of accessibility the image is representing the build tool. It doesn't seem necessary to announce that it's the logo.

**Comment by jakearchibald** [comment]
I'm happy if @argyleink & @una are happy.

**Review by argyleink — APPROVED** [review]

**Comment by jakearchibald** [comment]
Thanks @pajaydev! This change will be on the live site shortly

**Pull request merged**